tool/site/client/src/components/claim/createDispenseClaim.ts

```diff
@@ -289,3 +289,47 @@ function createEndorsementCodeableConcept(endorsement: EndorsementFormValues): f
   }
   return endorsementCodeableConcept
 }
+
+const tempPractitionerRole: fhir.PractitionerRole = {
```

Comment:
Temporary until this work is merged and Dan/Mukhtar can pull in these changes and adapt to their work on AEA-2325